### DISCUSSION TITLE
Link to a Group of Pages

### DISCUSSION THREAD

**Discussion by jon-whit** [facebook/docusaurus#4693] [Q&A]
I have some text that I'd like to link to a group of pages. For example

My Pages
.. Page 1
.. Page 2
  
  
Then something like `<a href="./docs/mypages">mypages</a>`. The idea being that I don't necessarily want to link a user directly to a page but instead just point them to the group/section.

How do I accomplish this? I'm not coming up with any good solutions from my research so far :)

**Comment by slorber** [comment]
> The idea being that I don't necessarily want to link a user directly to a page but instead just point them to the group/section.

There is no such concept as linking to a group of page. 

In the end you have to link to an URL, and that URL display a page.

The best you can do is link to the "first page" of your group of pages, but I don't even know what you mean by group of page, and how you define your groups.

How Docusaurus is supposed to know those pages are related in the first place?

If you mean docs (not pages), we could allow you to link to a sidebar or a sidebar category, as those are the only way to somehow group pages that I know of.

Later we'll introduce sidebar category pages, so that you can somehow create an "index page" for the category. It has been requested quite often.

**Reply by jon-whit** [reply]
@slorber  Sorry, I am new to the terminology (and docusaurus in general). I may have misspoke.

I think your statement 

> introduce sidebar category pages, so that you can somehow create an "index page" for the category. It has been requested quite often.

that is what I am looking for.

I have a sidebar category that "groups" pages.

docs/my-category/_category_.json
docs/my-category/page1.md
docs/my-category/page2.md

I want to link to "my-category" instead of having to link to a page within the group.

**Reply by slorber** [reply]
Issue to track this feature: https://github.com/facebook/docusaurus/issues/2643
